### PR TITLE
add mtls header base configuring doc

### PR DESCRIPTION
**Pull request by tharindu1st**
## Purpose
Update mtls releated doc

### CODE REVIEW

File: en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ This section explains how to APIs in WSO2 API Manager can be secured using mutua
 
 -   [Create an API Secured with Mutual SSL](#SecuringAPIswithMutualSSL-CreateanAPISecuredwithMutualSSL)
 -   [Invoke an API Secured with Mutual SSL from the Developer Portal](#SecuringAPIswithMutualSSL-InvokeanAPIsecuredwithMutualSSLfromtheDeveloperPortal)
```
Comment (bhathiya) [inline en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md:8]:
```suggestion
-   [Invoke an API Secured with Mutual SSL from Postman](#invoke-an-api-secured-with-mutual-ssl-using-postman)
```

File: en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ This section explains how to APIs in WSO2 API Manager can be secured using mutua
 
 -   [Create an API Secured with Mutual SSL](#SecuringAPIswithMutualSSL-CreateanAPISecuredwithMutualSSL)
```
Comment (bhathiya) [inline en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md:7]:
```suggestion
-   [Create an API Secured with Mutual SSL](#invoke-an-api-secured-with-mutual-ssl-using-postman)
```

File: en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md [hunk 3]
```diff
@@ -51,3 +51,28 @@ Listed below are the known limitations for this feature.
 -   Resource level security will not be applicable for the APIs that are only protected with mutual SSL.
 
 -   Scope level security will not be applicable for the APIs that are only protected with mutual SSL.
+### Invoke an API secured with Mutual SSL though SSL Terminated Load Balancer.
```
Comment (bhathiya) [inline en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md:54]:
```suggestion
### Handling MTLS when SSL is terminated by the loadbalancer/reverse proxy.
```

Comment (bhathiya) [reply en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md:54]: Also fix the link

File: en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md [hunk 4]
```diff
@@ -51,3 +51,28 @@ Listed below are the known limitations for this feature.
...
+
+When SSL Termination of invocation request happens at the loadbalancer, following prerequisites need to be met from the load balancer.
```
Comment (bhathiya) [inline en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md:56]:
```suggestion
When SSL Termination of invocation request happens at the loadbalancer/reverse proxy, the following prerequisites need to be met from the load balancer.
```

File: en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md [hunk 5]
```diff
@@ -51,3 +51,28 @@ Listed below are the known limitations for this feature.
...
+
+-   Terminate the mutual SSL connection from client
+-   Offload the Terminated SSL certficate from client into the Gateway in a Header.(eg: https://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_client_certificate)
```
Comment (bhathiya) [inline en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md:59]:
```suggestion
-   Pass the client SSL certificate to the Gateway in an HTTP Header. ( eg: https://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_client_certificate )
```

File: en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md [hunk 6]
```diff
@@ -51,3 +51,28 @@ Listed below are the known limitations for this feature.
...
+
+Below diagram show how MutualSSL works in such environment.
+
+![]({{base_path}}/assets/img/learn/mtls-loadbalancer.png)
+
+Default WSO2 API Manager supports to retrieve Certificate from **X-WSO2-CLIENT-CERTIFICATE** header. In order to change the retrival header,
+follow the below instructions.
```
Comment (bhathiya) [inline en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md:66]:
```suggestion
By default, the WSO2 API Manager retrieves the client certificate from the **X-WSO2-CLIENT-CERTIFICATE** HTTP header. In order to change the header,
```

File: en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md [hunk 7]
```diff
@@ -51,3 +54,27 @@ Listed below are the known limitations for this feature.
 -   Resource level security will not be applicable for the APIs that are only protected with mutual SSL.
 
 -   Scope level security will not be applicable for the APIs that are only protected with mutual SSL.
+### Handling MTLS when SSL is terminated by the loadbalancer/reverse proxy.
```
Comment (bhathiya) [inline en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md:57]: need a new line above this i think.

File: en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md [hunk 8]
```diff
@@ -51,3 +54,27 @@ Listed below are the known limitations for this feature.
...
+
+When SSL Termination of invocation request happens at the loadbalancer/reverse proxy,the following prerequisites need to be met from the load balancer.
```
Comment (bhathiya) [inline en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md:59]:
```suggestion
When SSL Termination of API requests happens at the loadbalancer/reverse proxy, the following prerequisites need to be met from the load balancer.
```

File: en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md [hunk 9]
```diff
@@ -51,3 +54,27 @@ Listed below are the known limitations for this feature.
...
+
+-   Terminate the mutual SSL connection from client
+-   Pass the client SSL certificate to the Gateway in an HTTP Header. ( eg: https://nginx.org/en/docs/http/ngx_http_ssl_module.html#ssl_client_certificate )
+
+Below diagram show how MutualSSL works in such environment.
```
Comment (bhathiya) [inline en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md:64]:
```suggestion
The below diagram shows how MutualSSL works in such environment.
```

File: en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md [hunk 10]
```diff
@@ -51,3 +54,27 @@ Listed below are the known limitations for this feature.
...
+
+![]({{base_path}}/assets/img/learn/mtls-loadbalancer.png)
+
+By default, the WSO2 API Manager retrieves the client certificate from the **X-WSO2-CLIENT-CERTIFICATE** HTTP header. In order to change the header,
+
+-  Navigate to the `<API-M_HOME>/repository/conf/deployment.toml` file.
+-  Configure the *certificate_header* under the [apimgt.mutual_ssl].
```
Comment (bhathiya) [inline en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md:71]:
```suggestion
-  Configure the *certificate_header* under the [apimgt.mutual_ssl] configuration. 
```

File: en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md (hunk 7 above)
Comment (bhathiya) [inline en/docs/learn/api-security/api-authentication/secure-apis-using-mutual-ssl.md:57]:
```suggestion
### Handling MTLS when SSL is terminated by the loadbalancer/reverse proxy.
```
```suggestion

### Handling MTLS when SSL is terminated by the loadbalancer/reverse proxy.
```

**Comment by Mariangela** [comment]
@tharindu1st , shouldn't this change go to master as well?

**Pull request merged**